avaframe/runCom1DFA.py

```diff
@@ -24,8 +24,8 @@
 
 # Start logging
 log = logUtils.initiateLogger(avalancheDir, logName)
-log.info('MAIN SCRIPT')
-log.info('Current avalanche: %s', avalancheDir)
+log.debug('MAIN SCRIPT')
+log.debug('Current avalanche: %s', avalancheDir)
```

Comment:
I think these should stay in Info, as this is an important info...

Reply:
After loading the configuration in com1DFA there is an info about 'Your current avalanche...' so this is why I removed this here, but changed it back to be info

Reply:
ahhh, sorry, forgot about that one. Then keep the debug level (since config *has* to be read anyway)

Reply:
ok I have set it back to debug :)